serverlessworkflow/modules/ROOT/pages/tooling/kn-plugin-workflow/kn-plugin-workflow-overview.adoc

```diff
@@ -0,0 +1,89 @@
+= Workflow plugin for Knative CLI
+
+`kn-plugin-workflow` is a plugin of the Knative Client, to enable users to quickly set up a local Kogito Serverless Workflow project from the command line.
+
+== Getting Started
+
+Note: In order to use the `workflow` plugin, you must install:
+
+- {java_install}[Java] at least version {java_min_version}
+- {maven_install}[Maven] at least version {maven_min_version}
+- {docker_install}[Docker] (Optional)
+- {podman_install}[Podman] (Optional)
+- {kubectl_install}[Kubernetes CLI].
+
+== Instalation
+
+You can download the latest binaries from the [Releases](https://github.com/kiegroup/kie-tools/releases) page.
+
+There are two ways to run `kn workflow`:
+
+1. You can run it standalone, just put it on your system path and make sure it is executable.
```

Comment:
maybe use the .Procedure here

Reply:
I'm not sure about using the ".Procedure" here, as we have "two ways to run".

Reply:
it is a adoc thing to help organizing the steps to do some action.
@hmanwani-rh can help with it, you can find examples in the other guides.